modules/sun/src/index.js

```diff
@@ -0,0 +1 @@
+export {getSolarPosition, getSunlightDirection} from './suncalc';
```

Comment:
Audit nit: Somehow it would be nice if these two names were more aligned - getSunPosition, getSunDirection....